src/translations/en.json

```diff
@@ -125,6 +125,10 @@
         "high": "high",
         "low": "low",
         "mode": "Mode",
+        "fan": "Fan",
+        "swing": "Swing",
+        "vertical_swing": "Vertical swing",
+        "horizontal_swing": "Horizontal swing",
```

Comment:
These translations are provided by the backend, they should not be duplicated here